input/pyilc_input_example_general.yml

```diff
@@ -106,6 +114,16 @@ beam_FWHM_arcmin: [1.4, 1.4] #update with whatever beam FWHM was used in the sim
 # a list of files containing the 1-D beam profiles to deconvolve from each map (only if beam_type is '1Dbeams') for the same order as freqs_delta_ghz
 beam_files: []
 
+#  wavelet_beam_criterion sets a number which is related to how much smaller than the needlet filters the beam of a certain frequency channel should be
+# in order to be dropped from the NILC in this needlet filter. If unspecified, defaults to 1e-3
+wavelet_beam_criterion: 0.001
+
+# override_N_freqs_to_use OVERRIDES wavleet_beam_criterion by explicitly specifying how many frequencies to use in each needlet scale in the NILC
+# it should be a list of ints of length N_scales
+# override_N_freqs_to_use: [2,2,2,2,2,2,2]
```

Comment:
but how does the user specify exactly *which* frequencies are used? (i.e., the number of frequencies alone is not sufficient info)

Reply:
It’s always the N highest resolution maps, so the last ones in the list (recall the list is from low resolution to high resolution). Functionality to allow to choose different subsets would be nice, but is not currently there.